paris/src/main/java/com/airbnb/paris/proxies/ViewProxy.java

```diff
@@ -214,6 +224,32 @@ public void setBackground(@Nullable Drawable drawable) {
         getView().setBackground(drawable);
     }
 
+    @Attr(R2.styleable.Paris_View_android_backgroundTint)
+    public void setBackgroundTint(@ColorRes int colorRes) {
```

Comment:
You should change the parameter type to `ColorStateList` here, that way the style builder for this attribute will also generate methods for `@ColorInt` and `ColorStateList`.